helm/values.yaml

```diff
@@ -0,0 +1,338 @@
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+  # podSecurityContext:
+  # fsGroup: 2000
+
+  podAnnotations: {}
+
+  # env - environment variables for the consumer API container
+  #############################################################################
+  # IMPORTANT
+  # The following environment variables are required:
+  # - Authentication__JwtSigningCertificate
+  # - Modules__Challenges__Infrastructure__SqlDatabase__ConnectionString
+  # - Modules__Devices__Infrastructure__AzureNotificationHub__ConnectionString
+  # - Modules__Devices__Infrastructure__SqlDatabase__ConnectionString
+  # - Modules__Files__Infrastructure__SqlDatabase__ConnectionString
+  # - Modules__Messages__Infrastructure__SqlDatabase__ConnectionString
+  # - Modules__Quotas__Infrastructure__SqlDatabase__ConnectionString
+  # - Modules__Relationships__Infrastructure__SqlDatabase__ConnectionString
+  # - Modules__Synchronization__Infrastructure__SqlDatabase__ConnectionString
+  # - Modules__Tokens__Infrastructure__SqlDatabase__ConnectionString
+  #############################################################################
+  env: []
+
+  service:
+    type: "ClusterIP"
+    port: 80
+
+  nodeSelector: {}
+  tolerations: []
+  affinity: {}
+
+  autoscaling:
+    enabled: false
+    minReplicas: 1
+    maxReplicas: 100
+    targetCPUUtilizationPercentage: 80
+    targetMemoryUtilizationPercentage: ""
+
+  # httpRoute - the configuration of the HttpRoute for the consumer API
+  httpRoute:
+    # parentRefName - the name of the gateway this Route wants to be attached to
+    parentRefName: ""
+    # parentRefNamespace - # the namespace of the gateway this Route wants to be attached to
+    parentRefNamespace: ""
+    hostnames: []
+
+  # httpRoute - only applicable if .Values.global.provider is set to "GoogleCloud"; see https://cloud.google.com/kubernetes-engine/docs/how-to/ingress-configuration#direct_health for a description of `HttpRoute`s`
+  backendConfig:
+    healthCheck:
+      # checkIntervalSec - time from the start of one prober's check to the start of its next check
+      checkIntervalSec: 15
+      # timeoutSec - the amount of time that Google Cloud waits for a response to a probe
+      timeoutSec: 15
+      # healthyThreshold - the number of consecutive successful checks required to mark a backend as healthy
+      healthyThreshold: 1
+      # unhealthyThreshold - the number of consecutive failed checks required to mark a backend as unhealthy
+      unhealthyThreshold: 2
+
+admincli:
+  nameOverride: ""
+
+  configuration:
+    Database:
+      # Provider - possible values: "Postgres"/"SqlServer"
+      Provider: "Postgres"
+
+  image:
+    repository: "ghcr.io/nmshd/backbone-admin-cli"
+    tagOverride: ""
+    pullPolicy: "IfNotPresent"
+    imagePullSecrets: []
+
+  resources:
+    requests:
+      cpu: "200m"
+      memory: "128Mi"
+    limits:
+      cpu: "400m"
+      memory: "1Gi"
+
+  # securityContext:
```

Comment:
you shouldn't set the security context, you can do `securityContext: {}` as you did with `podAnnotations` and `affinity` and leave it to the user to apply a security context.

Reply:
this also counts for the other parts where you use securityContext and podSecurityContext.